src/daemon.rs

```diff
@@ -582,6 +602,20 @@ impl Daemon {
             .chain_err(|| "failed to parse txid")
     }
 
+    pub fn test_mempool_accept(
+        &self,
+        txhex: Vec<String>,
+        maxfeerate: Option<f32>,
+    ) -> Result<Vec<MempoolAcceptResult>> {
+        let params = match maxfeerate {
+            Some(rate) => json!([txhex, format!("{:.8}", rate)]),
```

Comment:
f32::EPSILON (the smallest difference representable above 1.0) works out to about 11 sats when we consider 1.0 to be 1 BTC. So it can only really reliably show 7 decimal places.

Which I guess is fine if we consider 0.00001 BTC /kvB (5 decimal places) to be the minimum. (1 sat per vB) In which case showing 8 decimal places is pointless.

I wonder what the type is in Core. Might be worth it to use that, and round it to 8 if we use f64 and 7 if we use f32.

Reply:
https://doc.rust-lang.org/std/primitive.f32.html#associatedconstant.EPSILON

`0.00000011920929` is the smallest increment of f32.

Reply:
https://doc.rust-lang.org/std/primitive.f64.html#associatedconstant.EPSILON

`0.00000000000000022204460492503131` is smallest for f64

Reply:
Or we could always just accept the sane sat/vB instead of BTC/kvB, and just slide the decimal place 5 to the left. lol

Reply:
testmempoolaccept in Core throws an error if you give it too many decimal places (e.g. from floating point rounding errors), so it needs to be truncated to something.

I picked 8.d.p out of habit, but yeah I'll check what Core's bounds actually are.

Reply:
it turns out 8.d.p is actually exactly Core's limit (it's parsed as a fixed point number with scale 8):
https://github.com/bitcoin/bitcoin/blob/c1223188e0a5fb11c3a1b9224511a49dc2f848ed/src/rpc/util.cpp#L82-L87
https://github.com/bitcoin/bitcoin/blob/c1223188e0a5fb11c3a1b9224511a49dc2f848ed/src/rpc/util.h#L105

Reply:
In that case you should use f64 so that 8 decimal points can be properly represented without truncating.